src/cpp/conf/rdesktop-dev.conf

```diff
@@ -31,12 +31,6 @@ r-resources-path=${CMAKE_CURRENT_SOURCE_DIR}/session/resources
 r-session-library=${CMAKE_CURRENT_BINARY_DIR}/r/R/packages/library
 r-session-package-archives=${RSTUDIO_DEPENDENCIES_DIR}/common
 
-# override r home and doc dir (to ensure we always run against the version
-# we built against and so we can pick them up even when we are launched
-# standalone in the debugger)
-r-home-dir-override=${LIBR_HOME}
```

Comment:
This change makes it possible to build RStudio against one version of R, but run it against a different version of R if so desired. (Basically, identical behavior to what we do in release builds.)